tests/test_build.py

```diff
@@ -31,8 +32,13 @@ def nonascii_srcdir(request, rootdir, sphinx_test_tempdir):
     # If supported, build in a non-ASCII source dir
     test_name = u'\u65e5\u672c\u8a9e'
     basedir = sphinx_test_tempdir / request.node.originalname
-    try:
+    # Windows with versions prior to 3.2 (I think) doesn't support unicode on system path
+    # so we force a non-unicode path in that case
+    if sys.platform == "win32":
+        if not (sys.version_info.major >= 3 and sys.version_info.minor >= 2):
+            return basedir / 'all'
         srcdir = basedir / test_name
```

Comment:
It seems this change causes an errors (see Travis!)
`srcdir` is defined only with `sys.platfrom == "win32"`. Is this correct?